tests/installation/grub_test.pm

```diff
@@ -0,0 +1,44 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "basetest";
+use testapi;
+
+sub run() {
+    my $self = shift;
+    up_and_down();
```

Comment:
i have tested the maxlin approach, test continue to fails only with this, since you trust the needles matching. i would be more generous with 10 seconds more, than rather then 10 fails